### PR TITLE
Document rasterized parameter in pcolormesh() explicitly

### LINKED ISSUE #17708: Mention rasterized option in more methods
We should mention rasterized in a few places in the docstrings of relavent methods.  Ie pcolormesh, contour etc.  This option is probably the number one reason I started to use matplotlib,  but it seems a lot of people don’t realize it exists.

### PR DESCRIPTION
**Pull request by philip-sparks**
Update _axes.py to include mentions of rasterization within pcolormesh fixes #17708

PyData sprint contribution to enhance the documentation of rasterization within the pcolormesh function.

### CODE REVIEW

**Comment by timhoffm** [comment]
Please check the flake8 errors and fix them.

**Review by philip-sparks — COMMENTED** [review]
Adding flake8 changes

**Review by philip-sparks — COMMENTED** [review]
Fixing according to flake8 standards.

**Comment by philip-sparks** [comment]
All checks have passed @timhoffm

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -5834,6 +5834,12 @@ def pcolormesh(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
            differences see :ref:`Differences between pcolor() and pcolormesh()
            <differences-pcolor-pcolormesh>`.
 
+        .. hint::
+            `In order to speed up image processing of a plot with a large
+            amount of points, pcolormesh allows the use of the `rasterized`
+            keyword. As an example, see
+            :ref https://matplotlib.org/gallery/misc/rasterization_demo.html`.
```
Comment (jklymak) [inline lib/matplotlib/axes/_axes.py:5841]:
```suggestion
```

I would just document the kwarg...

Comment (jklymak) [reply lib/matplotlib/axes/_axes.py:5841]: Also, skeptical this renders correctly ;-)

File: lib/matplotlib/axes/_axes.py [hunk 2]
```diff
@@ -5925,6 +5931,9 @@ def pcolormesh(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
         snap : bool, default: False
             Whether to snap the mesh to pixel boundaries.
 
+        rasterized: bool, optional
+            Force rasterized (bitmap) drawing in vector backend output.
```
Comment (jklymak) [inline lib/matplotlib/axes/_axes.py:5935]:
```suggestion
            Rasterize the pcolormesh before drawing.  Can be faster and make smaller files for large data sets.
```

**Review by jklymak — COMMENTED** [review]
Thanks, but don't know that this deserves a hint at the beginning, though linking the rasterization demo is nice.

**Review by philip-sparks — COMMENTED** [review]
Need to change for flake8 standards.

**Comment by QuLogic** [comment]
Please give your PRs more descriptive titles in the future.

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
Thanks, and congratulations on your first contribution to Matplotlib. We hope to see you again.

**Comment by tacaswell** [comment]
It was great to meet you at pydata @philip-sparks !

**Comment by philip-sparks** [comment]
Thank you @tacaswell @timhoffm , my first open source contribution! And it will not be my last.

**Pull request merged**